src/app/screens/swap/useSwap.test.ts

```diff
@@ -1,64 +1,7 @@
 import { Currency } from 'alex-sdk';
-
-// TODO tim: fix jest transpiler to be able to import xverse-core
-// for now, the function is copied in place.
-//
-// import { selectedTokenReducer } from './useSwap';
-
-type SelectedCurrencyState = {
-  to?: Currency;
-  from?: Currency;
-  prevTo?: Currency;
-  prevFrom?: Currency;
-};
-
-type Side = 'from' | 'to';
-
-function updateOppositeCurrencyIfSameAsSelected(state, { newCurrency, side }) {
-  switch (side) {
-    case 'from':
-      if (state.to !== newCurrency) {
-        return state.to;
-      }
-      if (state.to === newCurrency && state.prevTo !== newCurrency) {
-        return state.prevTo;
-      }
-      return undefined;
-    case 'to':
-      if (state.from !== newCurrency) {
-        return state.from;
-      }
-      if (state.from === newCurrency && state.prevFrom !== newCurrency) {
-        return state.prevFrom;
-      }
-      return undefined;
-    default:
-      return state.to;
-  }
-}
-const selectedTokenReducer: (
-  state: SelectedCurrencyState,
-  { newCurrency, side }: { newCurrency: Currency; side: Side },
-) => SelectedCurrencyState = (state, { newCurrency, side }) => {
-  switch (side) {
-    case 'from':
-      return {
-        ...state,
-        prevFrom: state.from,
-        from: newCurrency,
-        to: updateOppositeCurrencyIfSameAsSelected(state, { newCurrency, side }),
-      };
-    case 'to':
-      return {
-        ...state,
-        prevTo: state.to,
-        to: newCurrency,
-        from: updateOppositeCurrencyIfSameAsSelected(state, { newCurrency, side }),
-      };
-    default:
-      return state;
-  }
-};
+import { describe, expect, test } from 'vitest';
+import { Side } from './types';
+import { selectedTokenReducer } from './useSwap';
```

Comment:
https://linear.app/xverseapp/issue/ENG-2474/fix-useswap-test-file-and-jest-unit-tests-in-general